deployment/ansible/deploy.yml

```diff
@@ -166,6 +166,14 @@
         when: genesis_accounts is defined
         tags: [config]
 
+       - name: "Copy node keys to {{ project_root }}/{{ config.keypair.dir }}"
+         copy:
+           src: "{{ item }}"
+           dest: "{{ project_root }}/{{ config.keypair.dir }}"
+           mode: 0600
+         with_items: "{{ key_pairs }}"
+         when: key_pairs is defined
```

Comment:
the variables is off, the chain keys are using `keypairs` ... what about `peer_keypairs`